fetchai/ledger/api/contracts.py

```diff
@@ -20,7 +21,7 @@ def create(self, owner: Entity, contract: 'Contract', fee: int, shard_mask: BitV
 
         # Default to wildcard shard mask if none supplied
         if not shard_mask:
-            print("WARNING: defaulting to wildcard shard mask as none supplied")
+            logging.warn("Defaulting to wildcard shard mask as none supplied")
```

Comment:
`logging.warn` is deprecated, use `logging.warning` instead.

```suggestion
            logging.warning("Defaulting to wildcard shard mask as none supplied")
```

Reply:
Done.